### ISSUE TITLE
File tree suddenly started appearing on the right, obscuring file content

### ISSUE THREAD

**Issue by msorens** [berzniz/github_pr_tree#127]
I have been using the extension for probably a year without incident, and it has always worked extremely well. Thank you!!

Suddenly today it started showing the file tree on the right and doing very odd things to the text.

Here's a screen shot on initial page load--notice file content is totally absent.
<img width="1434" alt="image" src="https://user-images.githubusercontent.com/6817500/93268607-4a157f80-f762-11ea-8b0f-f4ad7307b2b1.png">

If I then scroll down a bit, then the file content appears, but obscured by the plugin file tree.
(And it still seems to have allocated the empty slot on the left side of the page.)
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/6817500/93268695-7d580e80-f762-11ea-86f0-9b7a93da853b.png">

This is in Firefox v80.0.1; I do not believe there was any browser update that caused this.
I tried disabling then re-enabling the plugin; no change.
(Also, when it is disabled, the rest of the page looks and acts normal--no initial blank area like that shown above.)
I tried closing then re-opening Firefox; no change.

FYI: My repo is public; you can see the exact page above at https://github.com/chef/automate/pull/4330/files

**Comment by RussKie** [comment]
Got the same in Edge Version 85.0.564.51 (Official build).
My guess GubHub's changed markup again.

**Comment by lingzhang000** [comment]
Got the same error, agree that guess github changed markup again.

**Comment by StevenR152** [comment]
The same issue to in Chrome, Github is murdering this great tool with markup changes :-1:

**Comment by educalleja** [comment]
I love this extension.
I have just created a quick PR to fix the markup issue. Hopefully it is helpful.

**Comment by Klaster1** [comment]
As a temporary solution, I found this user CSS helps:
````
.enable_better_github_pr .__better_github_pr {
    position: absolute;
    top: 45px;
}

**Comment by dbadura** [comment]
The same issue for Firefox.

**Comment by berzniz** [comment]
Thank you all, a PR has been submitted. Once tested it will be released to the store

**Comment by berzniz** [comment]
Fix is in version 1.0.31.

Updates sent to both stores (Chrome and Firefox), hopefully will be reviewed and available to update soon.

**Comment by msorens** [comment]
👍  Update works for me! Thank you for such a quick fix!!

**Comment by p4bloch** [comment]
<3 THANK YOU!

**Comment by sidoruk-sv** [comment]
Hi, I know that updates are pushed and see that for Firefox `1.0.31` is already in marketplace
But for Chrome there is still a `1.0.30`:
https://chrome.google.com/webstore/detail/better-pull-request-for-g/nfhdjopbhlggibjlimhdbogflgmbiahc

Maybe there are some warnings while publishing?)

**Comment by berzniz** [comment]
The status is currently "Pending Review" on the Chrome Web Store.

The extension has been updated 30 times in the past, and the review time varied between hours to a week. I hope that with more active users, it jumps to the top of the queue.

**Comment by berzniz** [comment]
The new version with the fix (1.0.31) is now available on the Chrome Web Store.

You can force your browser to get the new version by going here `chrome://extensions/` and clicking `Update`.

I'm leaving this issue open for a few days so new users will see it and reduce overall noise.

Thanks for everyone who reported, submitted PRs and reviewed the code. You can head over to the Chrome/Firefox page and rate the extension to help it spread and have more people contributing.
